packages/client/src/__tests__/integration/happy/rejectNotFound/test.ts

```diff
@@ -50,21 +50,42 @@ for (const constructorKey of Object.keys(cases.constructor)) {
       const value = currentMethod[valueKey]
       test(`rejectOnNotFound | constructor=${constructorKey} | ${method}=${value}`, async () => {
         // It should fail or not
-        expect.assertions(1)
         const PrismaClient = await getTestClient()
         const prisma = new PrismaClient({
           rejectOnNotFound: constructor,
         })
 
-        // Test Rejection
-        try {
+        // This function name is important cause we're searching
+        // for it's name in the stack bellow
+        const testRejectionOnNotFound = async () => {
           const r = await prisma.user[method]({
             where: { id: 'none' },
             rejectOnNotFound: value,
           })
-          expect(r).toMatchSnapshot()
+
+          // If it got to here, then no error was thrown
+          // so we check the value if it's equal to null
+          expect(r).toBeNull()
+        }
+
+        try {
+          await testRejectionOnNotFound()
         } catch (error) {
-          expect(error).toMatchSnapshot()
```

Comment:
this was the line I wanted to suggest we should keep @YassinEldeeb in this comment: https://github.com/prisma/prisma/pull/10645/files#r768610234

Reply:
@capaj  yeah but @Jolg42 said that the snapshot would be different between the different OSs so It's not good to depend on the error snapshot, cause It won't be the same as for Windows/Linux/..etc

Reply:
Fair point, but I think the least we can do is:
`expect(error.message).toMatchSnapshot()`

Reply:
@YassinEldeeb I don't think the error snapshot in Jest includes the stack trace, but in case it does, you may use separate snapshots for `error.name` and `error.message`.

Reply:
(I'm pretty sure the error snapshot looks like just `[ErrorName: message]` without any platform-specific things though)

Reply:
yeah you are right: https://github.com/facebook/jest/blob/6d101e9b8190c0d265b40df96365dce704fc18d1/packages/pretty-format/src/__tests__/prettyFormat.test.ts#L109